backend/minors/BEM.yaml

```diff
@@ -0,0 +1,146 @@
+classroom and research experiences, enabling our students to gain 
+expertise and make important contributions to this critically 
+important field.
+urls: 
+- https://bioengineering.princeton.edu/undergraduate/bioengineering-minor
+contacts: 
+- type: Director of Undergraduate Studies
+  name: Jared Toettcher
+  email: toettcher@princeton.edu
+- type: Program Manager
+  name: Jessica Varela
+  email: jvarela@princeton.edu
+req_list: 
+- name: Prerequisites
+  max_counted: 1
+  min_needed: 2
+  explanation: |-
+    Students wishing to declare a Minor in Bioengineering must have taken 
+    a foundational course in Molecular Biology (MOL214 or equivalent) as 
+    well as one foundational course in computing (COS126 or equivalent).
+  pdfs_allowed: 0
+  completed_by_semester: 2
+  req_list:
+  - name: Molecular Biology
+    max_counted: 1
+    min_needed: 1
+    explanation: MOL214 or equivalent
+    double_counting_allowed: true
+    course_list: 
+    - MOL 214
+  - name: Computing
+    max_counted: 1
+    min_needed: 1
+    explanation: COS126 or equivalent
+    double_counted_allowed: true
+    course_list: 
+    - COS 126
+- name: Bioengineering Content
+  max_counted: 1
+  min_needed: 3
+  explanation: Three courses with bioengineering content, selected from the
+  list established for the Engineering Biology Certificate. These courses 
+  should provide a coherent training in an area of bioengineering, such as 
+  biotechnology, molecular or cellular engineering, neuroengineering, or 
+  systems biology. 
+  course_list: 
+  - CBE 262
+  - CBE 411
+  - CBE 419
+  - CBE 433
+  - CBE 438
+  - CBE 439
+  - CBE 440
+  - CBE 447
+  - CEE 385
+  - ENV 385
+  - CEE 477
+  - CHM 542
+  - MOL 542
+  - COS 343
+  - ECE 451
+  - BNG 451
+  - EEB 325
+  - EEB 330
+  - ELE 452
+  - ENE 318
+  - MAE 344
+  - MAE 416
+  - MAE 550
+  - MOL 433
+  - NEU 314
+  - NEU 330
+  - NEU 437
+  - NEU 443
+  - PSY 443
+  - NEU 457
+  - PHY 412
+  - PHY 562
+  - QCB 311
+  - QCB 408
+  - QCB 455
+- name: Restrictions
+  max_counted: 0
+  min_needed: 1
+  explanation: |-
+  One of these courses must be from outside the student's department of 
+  concentration, and at least one of these courses must not count as 
+  a departmental. 
+- name: Advanced Life Science Course
+  max_counted: 1
+  min_needed: 1
+  explanation: One advanced life science course, selected from the approved 
+  list. This course should provide additional insight into complex living 
+  systems and complement the bioengineering courses chosen by the student.
+  course_list: 
+  - ANT 405
+  - CEE 471
+  - EEB 314
+  - EEB 324
+  - EEB 403
+  - EEB 409
+  - GEO 417
+  - MOL 342
+  - MOL 345
+  - MOL 348
+  - MOL 380
+  - MOL 415
+  - MOL 423
+  - NEU 202
+  - NEU 422
+  - WWS 354
+- name: Restrictions
+  max_counted: 0
+  min_needed: 1
+  explanation: |-
+  Up to 2 courses can be double counted with a student's major
+- name: Ethical Aspects
```

Comment:
Let's add a no_req requirement and an editor's note in the explanation. For an example, check out [URB.yaml](https://github.com/HoagieClub/plan/blob/3f6b6d2f530c825d6cbc8c43d103d1001a5ef9ba/backend/certificates/URB.yaml#L340-L355)